### ISSUE TITLE
Disabled .custom-select isn't styled differently from enabled select

### ISSUE THREAD

**Issue by lenberman** [twbs/bootstrap#19365]
I add the disabled property programatically : the select is disabled, but the weight of the text does not change.

**Comment by patrickhlauke** [comment]
Please provide a JSBin, JSFiddle or similar isolated test case showing the problem.

**Comment by lenberman** [comment]
I have created a fiddle and  an example on the web.  The code is the same; however, the appearance is very different.  In [jsfiddle ()](https://jsfiddle.net/zdkdvjvz/2/) things behave as you would expect.  In the [online example](http://www.outtopastureenterprises.com/exper/form2.html), I see no difference between the disabled/enabled select.  Hope this is helpful.

**Comment by twbs-lmvtfy** [comment]
Hi @lenberman!

You appear to have posted a live example (https://fiddle.jshell.net/zdkdvjvz/2/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 31, column 7 thru column 54: No space between attributes.
* line 31, column 7 thru column 64: Quote ```` " ```` in attribute name. Probable cause: Matching quote missing somewhere earlier.
* line 31, column 7 thru column 258: Bad value ```` <link rel= ```` for attribute ```` src ```` on element ```` script ````: Illegal character in path segment: ```` < ```` is not allowed.
* line 31, column 7 thru column 258: Attribute ```` stylesheet" ```` not allowed on element ```` script ```` at this point.
* line 31, column 7 thru column 258: Attribute ```` href ```` not allowed on element ```` script ```` at this point.
* line 31, column 261 thru column 269: The text content of element ```` script ```` was not in the required format: Expected space, tab, newline, or slash but found ```` " ```` instead.
* line 35, column 7 thru column 60: No space between attributes.
* line 35, column 7 thru column 67: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 68: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 89: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 94: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 99: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 106: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 112: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 115: A slash was not immediately followed by ```` > ````.
* line 35, column 7 thru column 128: Quote ```` " ```` in attribute name. Probable cause: Matching quote missing somewhere earlier.
* line 35, column 7 thru column 129: Bad value ````     <script src= ```` for attribute ```` src ```` on element ```` script ````: Illegal character in path segment: ```` < ```` is not allowed.
* line 35, column 7 thru column 129: Attribute ```` https: ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` cdnjs.cloudflare.com ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` ajax ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` libs ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` tether ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` 1.1.1 ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` js ```` not allowed on element ```` script ```` at this point.
* line 35, column 7 thru column 129: Attribute ```` tether.min.js" ```` not allowed on element ```` script ```` at this point.
* line 35, column 141 thru column 149: Stray end tag ```` script ````.
* line 39, column 7 thru column 60: No space between attributes.
* line 39, column 7 thru column 67: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 68: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 92: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 102: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 116: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 119: A slash was not immediately followed by ```` > ````.
* line 39, column 7 thru column 135: Quote ```` " ```` in attribute name. Probable cause: Matching quote missing somewhere earlier.
* line 39, column 7 thru column 244: Bad value ````     <script src= ```` for attribute ```` src ```` on element ```` script ````: Illegal character in path segment: ```` < ```` is not allowed.
* line 39, column 7 thru column 244: Attribute ```` https: ```` not allowed on element ```` script ```` at this point.
* line 39, column 7 thru column 244: Attribute ```` maxcdn.bootstrapcdn.com ```` not allowed on element ```` script ```` at this point.
* line 39, column 7 thru column 244: Attribute ```` bootstrap ```` not allowed on element ```` script ```` at this point.
* line 39, column 7 thru column 244: Attribute ```` 4.0.0-alpha.2 ```` not allowed on element ```` script ```` at this point.
* line 39, column 7 thru column 244: Attribute ```` js ```` not allowed on element ```` script ```` at this point.
* line 39, column 7 thru column 244: Attribute ```` bootstrap.min.js" ```` not allowed on element ```` script ```` at this point.
* line 39, column 256 thru column 264: Stray end tag ```` script ````.
* line 43, column 3 thru column 25: Element ```` style ```` not allowed as child of element ```` body ```` in this context. (Suppressing further errors from this subtree.)
* line 47, column 3 thru column 9: Element ```` title ```` not allowed as child of element ```` body ```` in this context. (Suppressing further errors from this subtree.)
* line 74, column 1 thru column 7: Stray end tag ```` head ````.
* line 76, column 1 thru column 6: Start tag ```` body ```` seen but an element of the same type was already open.
* line 77, column 3 thru column 8: Start tag ```` body ```` seen but an element of the same type was already open.
* line 82, column 5 thru column 13: Element ```` formset ```` not allowed as child of element ```` form ```` in this context. (Suppressing further errors from this subtree.)
* line 95, column 1 thru column 7: Saw an end tag after ```` body ```` had been closed.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Deleted the redundant `<body>`: https://jsfiddle.net/cvrebert/zdkdvjvz/4/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (https://fiddle.jshell.net/cvrebert/zdkdvjvz/4/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 80, column 5 thru column 13: Element ```` formset ```` not allowed as child of element ```` form ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Assuming `<fieldset>` was intended: https://jsfiddle.net/cvrebert/zdkdvjvz/5/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (https://fiddle.jshell.net/cvrebert/zdkdvjvz/5/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 77, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Revised: https://jsfiddle.net/cvrebert/zdkdvjvz/6/

**Comment by iatek** [comment]
There seems to be another issue with the height of `select` when trying to use control sizing...

http://codeply.com/go/fy1CWmz03e

**Comment by mdo** [comment]
This was addressed in #19735.
